samples/manifests/plugins/dev-env-single-team.yaml

```diff
@@ -8,8 +8,6 @@ UserPoolId: !SSM ${/orbit-foundation/dev-env/resources::UserPoolId}
 SharedEfsFsId: !SSM ${/orbit-foundation/dev-env/resources::SharedEfsFsId}
 SharedEfsSgId: !SSM ${/orbit-foundation/dev-env/resources::SharedEfsSgId}
 InstallSsmAgent: true
-CognitoExternalProvider: !ENV ${COGNITO_EXTERNAL_PROVIDER::}
```

Comment:
same here .. don't change them.  Ramon will put a fix that if its empty its not doing it

Reply:
Added cognito attributes to manifest files.